content/nmdc/src/reference/metag_wrapper_doc.rst

```diff
@@ -0,0 +1,97 @@
+
+
+Workflow Availability
+---------------------
+The workflow is available in its individual components on GitHub and as a whole to run on `NMDC-EDGE <https://nmdc-edge.org/home>`_. 
+
+- `ReadsQC <https://github.com/microbiomedata/ReadsQC>`_ 
+- `MetaAssembly <https://github.com/microbiomedata/metaAssembly>`_
+- `MG Annotation <https://github.com/microbiomedata/mg_annotation>`_
+- `MetaMAGs <https://github.com/microbiomedata/metaMAGs>`_ 
+- `Read-based Taxonomy <https://github.com/microbiomedata/ReadbasedAnalysis>`_
+
+
+Requirements for Execution:  
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+(recommendations are in **bold**)
+
+- WDL-capable Workflow Execution Tool (**Cromwell**)
+- Container Runtime that can load Docker images (**Docker v2.1.0.3 or higher**)
+
+Hardware Requirements:
+~~~~~~~~~~~~~~~~~~~~~~
+- Disk space: 258 GB for databases 
+- 86 GB RAM
+
+Workflow Dependencies
+---------------------
+
+Third party software:
+~~~~~~~~~~~~~~~~~~~~~
+
+Docker images containing third-party software are available in the repsective repositories. More information is available in the workflow repositories listed above. 
+
+- `microbiomedata/bbtools:38.96 <https://hub.docker.com/r/microbiomedata/bbtools>`_
+- `bryce911/smrtlink:12.0.0.177059`_
+- `staphb/racon:1.4.20`_
+- `staphb/minimap2:2.25`_
+- `staphb/samtools:1.18`_
+- `staphb/spades:4.0.0`_
+- `microbiomedata/img-omics:5.2.0 <https://hub.docker.com/r/microbiomedata/img-omics>`_
+- `microbiomedata/nmdc_mbin <https://hub.docker.com/r/microbiomedata/nmdc_mbin>`_
+- `microbiomedata/nmdc_mbin_vis:0.7.0 <https://hub.docker.com/r/microbiomedata/nmdc_mbin_vis>`_
+
+
+
+Sample dataset(s):
+~~~~~~~~~~~~~~~~~~
+
+Zymobiomics mock-community DNA control (SRR7877884); this dataset is ~7 GB.
+
+Input: 
+~~~~~~~~~~~~~~~~~~~~
+A JSON file containing the following
+#.	output file prefix
+#.  path to :code:`input_file` if interleaved file
+#.  paths to :code:`input_fq1` and :code:`input_fq2` non-interleaved paired-end reads 
+#.	input_interleaved (boolean)
+#.	RNA strand type (optional) either left blank, :code:`aRNA`, or :code:`non_stranded_RNA`
+
+
+Output:
+~~~~~~~
+Upon completion of the run, the NMDC EDGE interface provides users can view the results, which are grouped by individual workflow. 
+
+In addition to the workflow outputs are summary tables for each portion: 
+
+- ReadsQC: statistics and metrics, including the number of reads and bases before and after QC filtering.
+- Read-based taxonomy: summary tables and interactive Krona plots as visual outputs.
+- Assembly: summary statistics
+- Annotation: statistics for processed sequences, predicted genes, and general quality information
+- MAGs: summary section with information on binned and unbinned contigs, genome completeness, estimated contamination, and the number of genes present on all bins determined to be high quality or medium quality.
```

Comment:
I noticed that some of these end with a period and some do not. I recommend either ending all of them with a period or not ending any of them with a period.